llvm/utils/lit/tests/Inputs/shtest-glob/example_file2.txt

```diff
@@ -0,0 +1,2 @@
+## This is the second example file used for testing glob pattern matching.
+RUN: This is the second example file.
```

Comment:
```suggestion
```
same here - also rename it to have a different suffix.

Reply:
I wasn’t too sure when you suggested renaming the suffix. I assumed you meant renaming this file suffix to `.input` to match the other file, so I did that. However, if you intended for a different suffix, like `.data` or something else, I can make that change. Please let me know if you have a specific suffix in mind.

Reply:
Yes that's exactly what I meant. The suffix doesn't matter we just want two files matching the glob.